src/InfoViz/Native/ParallelCoordinates/index.js

```diff
@@ -881,6 +825,11 @@ const DEFAULT_VALUES = {
 
   selectionOpacityAdjustment: 1,
   polygonOpacityAdjustment: 1,
+
+  hoverIndicatorHeight: 10,
+  hoverIndicatorWidth: 7,
+
+  numberOfBins: 128,
```

Comment:
Yes, I set it to the max as a default, but it's set by asking the histogram 2d provider for the number of bins elsewhere.